Plugin.pm

```diff
@@ -1208,8 +1245,25 @@ sub _trackItem {
 		$item->{line1} .= ' (' . cstring($client, 'PLUGIN_QOBUZ_HIRES') . ')';
 	}
 
-	if ( $track->{work} ) {
-		$item->{work} = $track->{work};
+	# Enhancements to work/composer display for classical music (tags returned from Qobuz are all over the place)
+	if ( $track->{album}->{isClassique} ) {
+		if ( $track->{work} ) {
+			$item->{work} = $track->{work};
+		} else {
+			# Try to extract the work from the title
+			if ( $track->{composer}->{name} && index($track->{composer}->{name}, (split ':\s*', $track->{title})[0]) == -1 ) {
```

Comment:
The result of the `split` statement could potentially be empty, right, thus index `0` undefined. You should evaluate this before the conditions. It also prevents doing the splitting repeatedly. 

And please put a regex in `/:\s*/` for readability.

Reply:
Done. I haven't seen an empty track title, but you never know...